lib/util.js

```diff
@@ -66,9 +66,11 @@ const Util = module.exports = {
       // if there are conflicts, log debug output
       if (conflicts.length > 0) {
         _.each(conflicts, conflictKey => {
-          app.log.debug('The config setting', `${key}.${conflictKey}`, 'is defined by both',
-            packName, 'and the application. The', conflictKey,
-            'config defined in', packName, 'will be ignored.')
+          if (app.config[key] && _.isArray(app.config[key][conflictKey])){
```

Comment:
What do you think about this sentence : `The config setting i18n.resources is defined by both trailpack-core and the application. trailpack-core:i18n.resources will be merge with application settings.` My english is not good so I'd rather you check is correct before I push.